app/controllers/admin/rebate_forms_controller.rb

```diff
@@ -38,8 +38,12 @@ def show
     end
   end
 
+  def edit
+  end
+
   # PATCH/PUT /admin/rebate_forms/1
   def update
+    @rebate_form.update(fields: rebate_form_fields_params, updated_by: current_user.email)
```

Comment:
Does this have to be two calls? What happens if we combine this into one?

Reply:
@Br3nda it says, `Validation failed: Blob must exist`